added docs for router context (#167)
ThreeDotsLabs/watermill: docs/content/docs/messages-router.md

{{% /render-md %}}

A full list of standard plugins can be found in [message/router/plugin](https://github.com/ThreeDotsLabs/watermill/tree/master/message/router/plugin).

### Context

> Every message received by handler, have some extra useful values in the `context`:

Reviewer: ```suggestion
Each message received by handler holds some useful values in the `context`:
```
